airbyte-cdk/python/unit_tests/sources/declarative/incremental/test_per_partition_cursor_integration.py

```diff
@@ -251,82 +250,3 @@ def test_substream_without_input_state():
             StreamSlice(partition={"parent_id": "2", "parent_slice": {}, },
                         cursor_slice={"start_time": "2022-02-01", "end_time": "2022-02-28"}),
         ]
-
-
-def test_substream_with_legacy_input_state():
```

Comment:
After talking with @girarda, we realized that this test does not test what we want it to, since it says the input state on the stream explicitly, bypassing the actual way the input state gets set. 

The behavior we want to test is already covered in `test_connector_state_manager.py::test_initialize_state_manager`, where we test on initializing the state manager with a legacy input.